imaging/ml/ml_codelab/breast_density_auto_ml.ipynb

```diff
@@ -75,20 +73,7 @@
    "source": [
     "### Store TCIA Dataset in Cloud Healthcare API\n",
     "\n",
-    "The [TCIA REST API ](https://wiki.cancerimagingarchive.net/display/Public/TCIA+Programmatic+Interface+%28REST+API%29+Usage+Guide)  will be used to fetch the images. The TCIA requires an API key which can be retreived by following the instructions in the **Getting Started with the TCIA API** section of the previous link. Once you receive the API key, assign it below (**NOTE: TCIA does not support self-registration, so expect some turn-around time until you get the key**). "
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "colab": {},
-    "colab_type": "code",
-    "id": "LjzzbUQOc5l0"
-   },
-   "outputs": [],
-   "source": [
-    "tcia_api_key = \"MY_KEY\" #@param"
+    "The [TCIA CBIS-DDSM](https://cloud.google.com/healthcare/docs/resources/public-datasets/tcia-attribution/cbis-ddsm?hl=da)  will be used to fetch the images."
```

Comment:
We do want to switch to using the hosted dataset, but I suggest we do this in a separate PR.
Also, we'd probably want to heavily modify the rest of this section when we do this, since there's won't be a need to download the dataset anymore.

Reply:
i guess we'll have more discussions about it and it will be new PR. may be it's better to have separate thread in https://github.com/GoogleCloudPlatform/healthcare/issues.
originally i was trying to implement as close as possible to previous version, so now i have few questions:
1 - do we need to keep examples how to import dicoms from gcs?
2 - if so, do we need them as working code examples or as links to docs?
3 - 'heavily modify' is not actually quite specific? if you'll describe it in issue i'll follow this paths, or implement this modifications as part of my changes

Reply:
1: I'd say no. Keep it simple, just use the datasets hosted in CHC API.
3: There's a lot of code/text that can basically go away. E.g. store_tcia_in_hc_api.py and most of the "Store TCIA Dataset in Cloud Healthcare API" section. Instead we should talk a bit about the hosted datasets, pointing to https://cloud.google.com/healthcare/docs/resources/public-datasets/tcia for further info, and then just export from the CBIS-DDSM dataset into JPEG directly.

Lets start a thread in https://github.com/GoogleCloudPlatform/healthcare/issues as you suggested if you feel that would be helpful.

Reply:
https://github.com/GoogleCloudPlatform/healthcare/issues/433